xep.dtd

```diff
@@ -121,7 +121,7 @@ THE SOFTWARE.
   alt CDATA ''
   height CDATA ''
   width CDATA ''>
-<!ELEMENT link (#PCDATA)* >
+<!ELEMENT link (#PCDATA | cite)* >
```

Comment:
It's not adding random things. It allows for things like  

```
<link url='http://xmpp.org/extensions/xep-0198.html#resumption'>
  <cite>XEP-0198</cite>§ 5
</link>
```

I've successfully generated HTML and PDF versions using this.